src/e2e/kotlin/fi/aalto/cs/apluscourses/e2e/PlaceholderTest.kt

```diff
@@ -18,7 +18,9 @@ class PlaceholderTest {
 
   @Test
   fun mainTest() = uiTest {
+    //step 2
```

Comment:
nice idea 😉 (I assume these are the steps from [TESTING.md](https://github.com/Aalto-LeTech/intellij-plugin/blob/master/TESTING.md))?

Reply:
yes they are